specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/putMachineLearningServicesQuotaRequestDedicated.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "scope": "subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.MachineLearningServices/locations/eastus",
+    "resourceName": "StandardDv2Family",
+    "api-version": "2021-03-15",
+    "createQuotaRequest": {
+      "properties": {
+        "limit": 200,
+        "unit": "Count",
+        "name": {
+          "value": "StandardDv2Family"
+        },
+        "resourceType": "dedicated"
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Quota/resourceProviders/Microsoft.MachineLearningServices/locations/eastus/serviceLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "name": "2B5C8515-37D8-4B6A-879B-CD641A2CF605",
```

Comment:
>2B5C8515-37D8-4B6A-879B-CD641A2CF605" [](http://example.com/codeflow?start=17&length=37)

The same comment for the resource name. It should 'StandardDv2Family' from the request.